traffic_editor/gui/building_level.cpp

```diff
@@ -837,8 +837,38 @@ void BuildingLevel::draw(
         break;
       }
     }
-    if (pixmap.isNull())
-      continue;  // couldn't load the pixmap; ignore it.
+    if (pixmap.isNull()) {
+      // BACKWARDS COMPATIBILITY PATCH: Try again, but...
+      // Use the first namespaced thumbnail for a specified non-namespaced
+      // model with a warning instead. (Also modifies the model name inplace!)
+
+      for (auto &editor_model : editor_models)
+      {
+        // Check if non-namespaced name is a substring of namespaced name
+        // If yes, use it!
+        if (editor_model.name.find(model.model_name) != std::string::npos)
```

Comment:
We should compare the whole `editor_model.name`'s suffix with `model.model_name`, otherwise we might get cases of `table` getting paired with `OpenRobotics/cafe_table` instead of `OpenRobotics/table`

Reply:
for example,
```
[WARNING] Thumbnail table not found, substituting OpenRobotics/cafe_table instead!
(table will be saved as OpenRobotics/cafe_table!)

[WARNING] Thumbnail TrolleyBed not found, substituting OpenRobotics/SmartTrolleyBed instead!
(TrolleyBed will be saved as OpenRobotics/SmartTrolleyBed!)
```

Reply:
Fixed in https://github.com/osrf/traffic_editor/pull/136/commits/1052bcb7e6d2bc057a0adcd13dbf4801f4b6e1a2 !